lib/pdf_fill/filler.rb

```diff
@@ -48,5 +49,26 @@ def fill_form(saved_claim)
 
       combine_extras(file_path, hash_converter.extras_generator)
     end
+
+    def fill_ancillary_form(form_id, form_data, claim_id)
```

Comment:
are you guys not planning on using the `SavedClaim` model for form submission, and if so why not? using that model would save you the trouble of creating this method, and you would also have to make modify or duplicate `CentralMail::SubmitSavedClaimJob` to get it working with another model or just a form hash.

Reply:
@lihanli 

We are indeed planning to use the SavedClaim model for form submission.  I ran into an issue testing the PDF generation for this form (VA21-4142) because it is an ancillary form without its own claim.  It will be included in the submission of another claim, such as along with a 526.  

I saw a test for PDF generation in filler_spec.rb, and wanted to add 4142 to that test, but was unable to create a claim to pass in to the fill_form method.  That being the case, I created fill_ancillary_form specifically to be able to test the PDF generation for ancillary forms.  I realize this is not ideal, and I can move the method to a "form_spec_helper" module and use it just for unit tests from there.  There will be more forms down the line that will run up against this issue.

Do you have any pointers on how to create a test claim for this type of form, so I can use the existing filler_spec test instead?  If not, I will just move the method to the test area.